src/java.base/share/classes/jdk/internal/util/ByteArrayLittleEndian.java

```diff
@@ -132,6 +132,23 @@ public static int getInt(byte[] array, int offset) {
         return (int) INT.get(array, offset);
     }
 
+    /**
+     * {@return an {@code unsigned int} from the provided {@code array} at the given {@code offset}
+     * using little endian order}.
+     * <p>
+     * There are no access alignment requirements.
+     *
+     * @param array  to get a value from.
+     * @param offset where extraction in the array should begin
+     * @return an {@code int} representing an unsigned short from the array
+     * @throws IndexOutOfBoundsException if the provided {@code offset} is outside
+     *                                   the range [0, array.length - 4]
+     * @see #setUnsignedInt(byte[], int, long)
+     */
+    public static long getUnsignedInt(byte[] array, int offset) {
+        return Integer.toUnsignedLong((int) INT.get(array, offset));
+    }
```

Comment:
Hello Glavo, I was going to recommend adding a test method to existing jtreg tests to test these new methods. But it looks like there's no jtreg test for this `ByteArrayLittleEndian` class. Would you mind creating a new test class to (at least) test these methods?

Reply:
> Hello Glavo, I was going to recommend adding a test method to existing jtreg tests to test these new methods. But it looks like there's no jtreg test for this `ByteArrayLittleEndian` class. Would you mind creating a new test class to (at least) test these methods?

I think I could modify `test/jdk/jdk/internal/util/ByteArray/ReadWriteValues.java` to also test `ByteArrayLittleEndian`.

Reply:
I added a test for `ByteArrayLittleEndian` in #14636